hack/build-images.sh fails with "stat examples/gitserver/bin/gitserver: no such file or directory"

```
[INFO] hack/build-rpms.sh exited with code 0 after 00h 13m 06s
hack/build-images.sh
...
[openshift/origin-gitserver] --> FROM openshift/origin
[openshift/origin-gitserver] --> COPY bin/gitserver /usr/bin/gitserver
[openshift/origin-gitserver] --> COPY hooks/ /var/lib/git-hooks/
[openshift/origin-gitserver] --> COPY gitconfig /var/lib/gitconfig/.gitconfig
[openshift/origin-gitserver] --> RUN mkdir -p /var/lib/git &&     mkdir -p /var/lib/gitconfig &&     chmod 777 /var/lib/gitconfig &&     ln -s /usr/bin/gitserver /usr/bin/gitrepo-buildconfigs
[openshift/origin-gitserver] --> VOLUME /var/lib/git
[openshift/origin-gitserver] --> ENV HOME=/var/lib/gitconfig
[openshift/origin-gitserver] --> ENTRYPOINT ["/usr/bin/gitserver"]
[openshift/origin-gitserver] --> Committing changes to openshift/origin-gitserver:13d86e2 ...
[openshift/origin-gitserver] unable to commit build container: API error (500): {"message":"devmapper: Error mounting '/dev/mapper/docker-202:2-92453254-22d97f22cdbe261c12e113f6067b2ed5dd4179de0ce8bb3549d259e31aebef2a' on '/var/lib/docker/devicemapper/mnt/22d97f22cdbe261c12e113f6067b2ed5dd4179de0ce8bb3549d259e31aebef2a': invalid argument"}
[openshift/origin-gitserver] Removing examples/gitserver/bin/gitserver
[WARNING] Retrying image build for openshift/origin-gitserver:latest, attempt 1...
[openshift/origin-gitserver] --> FROM openshift/origin
[openshift/origin-gitserver] --> COPY bin/gitserver /usr/bin/gitserver
[openshift/origin-gitserver] stat examples/gitserver/bin/gitserver: no such file or directory
[ERROR] PID 30580: hack/lib/constants.sh:56: `return "${result}"` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/lib/constants.sh:56: `return "${result}"`
[INFO] 		  2: hack/build-images.sh:25: os::build::images
[INFO]   Exiting with code 1.
```
Full log is here: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/16934/test_pull_request_origin_end_to_end/5565/

--- a/hack/lib/constants.sh
+++ b/hack/lib/constants.sh
@@ -32,7 +32,6 @@ readonly OS_SDN_COMPILE_TARGETS_LINUX=(
 )
 readonly OS_IMAGE_COMPILE_TARGETS_LINUX=(
   cmd/dockerregistry
-  cmd/gitserver
   vendor/k8s.io/kubernetes/cmd/hyperkube
   "${OS_SDN_COMPILE_TARGETS_LINUX[@]}"
 )
@@ -344,7 +343,6 @@ readonly OS_ALL_IMAGES=(
   openshift/origin-egress-router
   openshift/origin-egress-http-proxy
   openshift/origin-recycler
-  openshift/origin-gitserver
   openshift/origin-cluster-capacity
   openshift/origin-service-catalog
   openshift/origin-template-service-broker
@@ -368,7 +366,6 @@ function os::build::images() {
 
   # Link or copy image binaries to the appropriate locations.
   ln_or_cp "${OS_OUTPUT_BINPATH}/linux/amd64/hello-openshift" examples/hello-openshift/bin
-  ln_or_cp "${OS_OUTPUT_BINPATH}/linux/amd64/gitserver"       examples/gitserver/bin
 
   # determine the correct tag prefix
   tag_prefix="${OS_IMAGE_PREFIX:-"openshift/origin"}"
@@ -394,7 +391,6 @@ function os::build::images() {
   # images that depend on "${tag_prefix}
   ( os::build::image "${tag_prefix}-haproxy-router"        images/router/haproxy ) &
   ( os::build::image "${tag_prefix}-keepalived-ipfailover" images/ipfailover/keepalived ) &
-  ( os::build::image "${tag_prefix}-gitserver"             examples/gitserver ) &
   ( os::build::image "${tag_prefix}-deployer"              images/deployer ) &
   ( os::build::image "${tag_prefix}-recycler"              images/recycler ) &
   ( os::build::image "${tag_prefix}-docker-builder"        images/builder/docker/docker-builder ) &

